editor/plugins/sprite_frames_editor_plugin.cpp

```diff
@@ -1302,13 +1306,15 @@ void SpriteFramesEditorPlugin::edit(Object *p_object) {
 	AnimatedSprite2D *animated_sprite = Object::cast_to<AnimatedSprite2D>(p_object);
 	if (animated_sprite) {
 		s = *animated_sprite->get_sprite_frames();
+		frames_editor->set_tree_texture_filter(animated_sprite->get_texture_filter());
```

Comment:
Probably needs to run a bit more logic, since by default it inherits the filter from the parent node.

Reply:
@IgorKordiukiewicz You can now use methods from #66546